noobaa/noobaa-core
# Move server-diagnostics-form into new arch

[skip ci]
Explain the changes:
Move server-diagnostics-form into the new architecture

Testing Instructions:
1. go to the server page
2. select 'diagnostics' tab
3. refresh the page, no errors found
4. click Turn On System Debug Mode button
5. debug mode enabled, time countdown 10 min
6. click Turn Off System Debug Mode
7. if debug mode enabled and the countdown is finished, page state should be updated

### Reminders:
- Create a new test - the first is the hardest
- User Experience is top priority
- Design for failure
- Keep it simple
- `npm test`
- Imagine a support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade - DB schema, server platform, agents install, npm packages
